src/core/load_balancing/pick_first/pick_first.cc

```diff
@@ -648,7 +648,8 @@ void PickFirst::HealthWatcher::OnConnectivityStateChange(
     case GRPC_CHANNEL_TRANSIENT_FAILURE:
       policy_->channel_control_helper()->UpdateState(
           GRPC_CHANNEL_TRANSIENT_FAILURE, status,
-          MakeRefCounted<TransientFailurePicker>(status));
+          MakeRefCounted<TransientFailurePicker>(absl::UnavailableError(
+              absl::StrCat("health watch: ", status.message()))));
```

Comment:
I think we should also make sure that this message includes the IP address.  However, I think we need to do that below this layer, not here in pick_first.

It looks like we're already prepending the address if the failure comes from the client-side health checking code:

https://github.com/grpc/grpc/blob/478c8850e2abbc7e5beb39486404d44d008a1116/src/core/load_balancing/health_check_client.cc#L173

However, we need to do the same thing if the failure comes from the outlier detection code:

https://github.com/grpc/grpc/blob/478c8850e2abbc7e5beb39486404d44d008a1116/src/core/load_balancing/outlier_detection/outlier_detection.cc#L174
https://github.com/grpc/grpc/blob/478c8850e2abbc7e5beb39486404d44d008a1116/src/core/load_balancing/outlier_detection/outlier_detection.cc#L195

Reply:
done